toolbox/index.md

```diff
@@ -216,11 +216,11 @@ Web Server: Javalin
 ![Javalin](javalin.png){:width="434"}
 </figure>
 
-**What it is.** A [Java](#programming-language-java) library for developing web servers.
+**What is it.** A [Java](#programming-language-java) library for developing web servers.
 
 **Our Choice.** Javalin abstracts the low-level aspects of the communication between server and client while avoiding abstractions that would obscure how things work and confuse beginners.
 
-**Where to Get it.** Add `io.javalin:javalin` as a dependency—the [build system](#build-system-gradle) takes care of the rest. Besides Javalin itself, also add a logger, [Simple Logging Facade for Java (SLF4J)](https://www.slf4j.org) (`org.slf4j:slf4j-simple`), so that Javalin can log important information while it’s running.
+**Where to Get it.** Add `io.javalin:javalin` as a dependency — the [build system](#build-system-gradle) takes care of the rest. Besides Javalin itself, also add a logger, [Simple Logging Facade for Java (SLF4J)](https://www.slf4j.org) (`org.slf4j:slf4j-simple`), so that Javalin can log important information while it’s running.
```

Comment:
Hmmmm, spaces around dashes are a matter of style, so I’m not taking them. That’s what [CMOS does](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/HyphensEnDashesEmDashes/faq0108.html), for example.